components/expenses/ExpenseDetails.js

```diff
@@ -333,37 +335,34 @@ class ExpenseDetails extends React.Component {
                     className="amountField"
                     onChange={amount => this.handleChange('amount', amount)}
                   />
-                )}
-                {!(editMode && canEditAmount) && (
-                  <FormattedNumber value={expense.amount / 100} currency={expense.currency} {...this.currencyStyle} />
-                )}
```

Comment:
The amount is already displayed at the top right of the expense, there was no point showing it there outside of edit mode.